examples/topics/categorical/correlograms.py

```diff
@@ -0,0 +1,68 @@
+''' A categorical plot showing the correlations in mineral content for 214 samples of glass fragments
+obtained during forensic work. The dataset contains seven variables measuring the amounts of magnesium (Mg),
+calcium (Ca), iron (Fe), potassium (K), sodium (Na), aluminum (Al), and barium (Ba) found in each glass fragment.
+The colored circles represent the correlations between pairs of these variables.
+The magnitude of each correlation is encoded in the size of the circles.
+
+.. bokeh-example-metadata::
+    :sampledata: forensic_glass
+    :apis: bokeh.plotting.figure.scatter
+    :refs: :ref:`ug_topics_categorical_scatters`
+    :keywords: scatter, correlogram
+
+'''
+import numpy as np
+import pandas as pd
+
+from bokeh.models import ColorBar, ColumnDataSource, FixedTicker, LinearColorMapper
+from bokeh.plotting import figure, show
+from bokeh.sampledata.forensic_glass import data
+from bokeh.transform import transform
+
+df = data.copy()
+
+pairs = ["Mg-Ba", "Mg-Al", "Mg-Na", "Mg-K", "Mg-Fe", "Mg-Ca", "Ca-Ba",
+         "Ca-Al", "Ca-Na", "Ca-K", "Ca-Fe", "Fe-Ba", "Fe-Al", "Fe-Na",
+         "Fe-K", "K-Ba", "K-Al", "K-Na", "Na-Ba", "Na-Al", "Al-Ba"]
+
+pair_split = [pair.split("-") for pair in pairs]
+correlations = []
+for pair in pair_split:
+    matrix = np.corrcoef(df[f"{pair[0]}"], df[f"{pair[1]}"])
+    correlation = matrix[0, 1]
+    correlations.append(correlation)
```

Comment:
not sure the extra variable adds much here
```suggestion
    correlations.append(matrix[0, 1])
```